src/components/tests/negotiations-panel.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { createMount } from '@material-ui/core/test-utils';
+import React from 'react';
+import TreeView from '@material-ui/lab/TreeView';
+import TreeItem from '@material-ui/lab/TreeItem';
+import { NegotiationConflict, NegotiationStatus, ResolveState } from '../../negotiation-status-manager';
+import NegotiationsPanel from '../negotiations-panel';
+
+const mount = createMount();
+
+let negotiationStatuses : Record<number, NegotiationConflict>;
+
+beforeEach(() => {
+  negotiationStatuses = {
+    0: {
+      participantIdsToNames: { 
+        "1":"tinyrobot1",
+        "2":"tinyrobot2" 
+      },
+      participantIdsToStatus: { 
+        "1": {
+          hasTerminal: true,
+          base: { 
+            sequence: [1],
+            defunct: false,
+            rejected: true,
+            forfeited: false 
+          },
+          terminal: { 
+            sequence: [2, 1],
+            defunct: false,
+            rejected: false,
+            forfeited: false 
+          }
+        },
+        "2": {
+          hasTerminal: false,
+          base: { 
+            sequence: [2],
+            defunct: false,
+            rejected: false,
+            forfeited: true 
+          },
+          terminal: {
+            sequence: [1, 2],
+            defunct: true,
+            rejected: false,
+            forfeited: false 
+          }
+        },
+      },
+      resolved: ResolveState.RESOLVED
+    }
+  }
+});
+
+it('renders negotiations', () => {
+  const root = mount(<NegotiationsPanel 
+    conflicts={negotiationStatuses}
+    spotlight={undefined}
+    trajManager={undefined}
+    negotiationTrajStore={undefined} />);
+
+  const treeView = root.find(TreeView);
+  expect(treeView).toBeDefined();
+  
+  const treeItem = root.find(TreeItem);
+  expect(treeItem).toBeDefined();
+
+  root.unmount();
+});
```

Comment:
Maybe we can add a test to verify that the colors are being assigned correctly depending on the status of the negotiation. (you can leave it as a TODO if you want to)

You could also add a snapshot test with a fixed number of deals.

Reply:
add snapshot test c111ae7

Reply:
Since KP told us to reduce the number of the codebot warnings and augment the number of test, we should add tests to verify that the colors are being assigned correctly depending on the status of the negotiation, that the status labels are assigned correctly, etc.

Reply:
37460d2 
let me know of a way to get TreeItems at the next depth, else that's all i can do for now